src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.html

```diff
@@ -0,0 +1,22 @@
+<!-- Embed dashboard -->
+<!-- TODO: display loading spinner as long as the images haven't loaded -->
+<cd-info-panel>
+  <ng-container i18n *ngIf="grafanaExist">
+    Retrieving data might take some time. Please be patient.
```

Comment:
I'm curious about this -- do we really want to have this banner visible all the time, and *why* do we think that retrieving data might take some time, shouldn't fetching the data from prometheus be fast?

Reply:
Ack, will replace it with progress bar